hw4.Rmd

```diff
@@ -30,43 +36,71 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+data %>%
+  select(decision) %>%
+  mutate(decision.dummy = ifelse(decision == "no", 0,
+                          ifelse(decision == "yes", 1, NA))) %>%
+  summarise(prop.granted = mean(decision.dummy))
 ```
 
 (@) There are 12 different judges in the data. A key question is whether different judges have different rates of granting an appeal.  Make a plot showing the number of cases per judge.
 
 ```{r}
-# [your code here]
+data.prop.granted <- data %>%
+  select(decision, judge) %>%
+  mutate(decision.dummy = ifelse(decision == "no", 0,
+                          ifelse(decision == "yes", 1, NA))) %>%
+  group_by(judge) %>%
+  summarise(prop.granted = mean(decision.dummy)) %>%
+  arrange(prop.granted)
+
+data.prop.granted 
 ```
 
 (@) Now plot the rate at which the judges grant appeals. 
 
 ```{r}
-# [your code here]
+ggplot(data= data.prop.granted, aes(judge, prop.granted)) + geom_point()
 ```
 
 (@) Now let's try this with logistic regression because we are going to move to more complicated models.  Fit a logistic regression predicting whetheran appeal will be granted using judge as a categorical predictor. Use Iacobucci as the reference judge to facilitate the comparisons that follow. For more on how to control the reference level of a factor, check out this post on StackOverflow: http://stackoverflow.com/questions/3872070/how-to-force-r-to-use-a-specified-factor-level-as-reference-in-a-regression
 
 ```{r}
-# [your code here]
+data <- data %>%
+  mutate(decision.dummy = ifelse(decision == "no", 0,
+                          ifelse(decision == "yes", 1, NA)),
+         judge= as.factor(judge))
+data$judge <- relevel(data$judge, ref="Iacobucci")
+reg <- glm(decision.dummy ~ judge, family="binomial", data=data)
```

Comment:
It is easier to grade if you display the output of the regression in the html file.